### DISCUSSION TITLE
Orchestrate SqlServer storedProcedures with Dagster

### DISCUSSION THREAD

**Discussion by 54646546** [dagster-io/dagster#4437] [Q&A] [answered]
Would it be possible to use Dagster to orchestrate SqlServer storedProcedures ? :

We a complexe flow, with several "projects" depending on each other, and each project have several storedProcedure to run in a specific order...
We need to manage first the dependencies within a project, and start task parallely, but also manage the dependencies between the projects...

Would Dagster be suitable for that ?
Also seen on video, that the scheduler is not yet working on windows, is it still the cas ?

We would be also interested in commercial support and could found some needed devs for that use-case

**Comment by sryza** [comment, marked as answer]
The Dagster scheduler now works with windows - the video you watched must have been out of date.

it should be possible to orchestrate sql server stored procedures using Dagster, as long as there’s a way to invoke them from python or from the command line.

**Reply by yuhan** [reply]
Hi @54646546, just to add onto what Sandy said, the scheduler now works with Windows and here's some notes from a user on setting up Dagster as a service (including the scheduler) with Windows: https://github.com/dagster-io/dagster/discussions/7223

Besides, we have a commercial support Dagster Cloud. you can sign up here to join the early access program: https://dagster.io/cloud
